package.json

```diff
@@ -93,6 +92,7 @@
     }
   },
   "devDependencies": {
+    "@balena/lint": "^4.1.1",
```

Comment:
Yes it's 5.x that will drop node 8 support, that said it would only affect developing on the CLI with node 8 and not running it, so imo that would be fine to do without a major (but a major dropping node 8 would be really nice as it unlocks a bunch of stuff for the cli to use)